src/scripts/public/dom-loader.js

```diff
@@ -36,7 +36,11 @@ const domReady = () => {
 			nav = header.querySelector('nav');
 		}
 
-		if (null !== window.site.siteName) {
+		if (
+			null !== window.site.siteName &&
```

Comment:
only if javascript had an equivalent to php's empty, I don't know if lodash has any function to help here. ....

Reply:
I'll look into loadash isEmpty for future use. Could also roll our own version with a little effort. It'd be nice not to have to constantly joust with javascript's undefined...'undefined'...null...0...""...falsy values.